subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/UpToDateScalaCompileIntegrationTest.groovy

```diff
@@ -110,7 +109,6 @@ class UpToDateScalaCompileIntegrationTest extends AbstractIntegrationSpec {
     }
 
     def "compilation emits toolchain usage events"() {
-        def operations = new BuildOperationsFixture(executer, temporaryFolder)
```

Comment:
You could have an `captureBuildOperations` in your fixture, which would init the operations field. And think that would be better so not all tests capture build operations.